start_provider.sh

```diff
@@ -0,0 +1,16 @@
+# Export env vars
+export PROVIDER_CONFIG_FILE=config.ini
```

Comment:
I also commited a version of this. I agree that it is very useful for development. Can you please remove exportlocal.sh (my version of this) when you go through this review?

Reply:
Done.  (I can't remember if or when I removed it, but it appears to be gone already.) Thanks for the comment!